client/ayon_flame/api/scripts/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""AYON Autodesk Flame api/scripts
```

Comment:
Doesn't the flame logic go through all files in scripts folder and executes them? In other words, doesn't this trigger the same logic twice because it does import the code, again?

Reply:
Hum.. good call indeed.
I've actually removed this file because discussing with Jakub we went with a different approach.
But thanks for the flag I'll definitely keep that in mind.